app/src/main/java/com/parishod/watomatic/model/preferences/PreferencesManager.java

```diff
@@ -8,6 +8,8 @@
     private final String KEY_SERVICE_ENABLED = "pref_service_enabled";
     private final String KEY_GROUP_REPLY_ENABLED = "pref_group_reply_enabled";
     private final String KEY_AUTO_REPLY_THROTTLE_TIME_MS = "pref_auto_reply_throttle_time_ms";
+    private final String KEY_WHATSAPP_ENABLED = "pref_whatsapp_enabled";
+    private final String KEY_FACEBOOK_MESSENGER_ENABLED = "pref_facebook_enabled";
```

Comment:
lets save enabled package names in an array if possible:
Example if only facebook is enabled:
Key: `auto_reply_enabled_arr`
Value: `{"com.facebook.mlite", "com.facebook.orca"}`

Reply:
I see that we have to add two packages for facebook but thats okay I guess. When FB is enabled, both should be added and when removed both should be removed. In future if we separate FB from FB lite, then it may be little easy